### PR TITLE
upgrade cam for 4.4

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1844627

For 4.4

### CODE REVIEW

File: modules/migration-upgrading-migration-tool.adoc [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+You can upgrade your Cluster Application Migration (CAM) tool on your source and target clusters.
+
+If you are upgrading from CAM 1.1 to 1.2, you must update the service account token in the CAM web console.
+
+[id='upgrading-cam-ocp-4_{context}']
+== Upgrading the CAM on an {product-title} 4 cluster
+
+You can upgrade CAM on an {product-title} 4 cluster with the Operator Lifecycle Manager.
+
+If you selected the *Automatic* approval option when you subscribed to the Cluster Application Migration Operator, the CAM is updated automatically.
```
Comment (lamek) [inline modules/migration-upgrading-migration-tool.adoc:17]: I would remove "the" or change this to "the CAM **tool**..."

File: modules/migration-upgrading-migration-tool.adoc [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+. In the {product-title} console, navigate to *Operators* > *Installed Operators*.
+. Click *Cluster Application Migration Operator*.
+. In the *Subscription* tab, change the *Approval* option to *Automatic*.
+. Optional: Edit the *Channel*.
++
+Updating the subscription deploys the updated Cluster Application Migration Operator and updates the CAM tool components.
+
+ifdef::migrating-3-4[]
+[id='upgrading-cam-ocp-3_{context}']
+== Upgrading the CAM on an {product-title} 3 cluster
```
Comment (lamek) [inline modules/migration-upgrading-migration-tool.adoc:32]: Same as above, remove "the" or add "the CAM **tool**"

File: modules/migration-upgrading-migration-tool.adoc [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+You can upgrade CAM on an {product-title} 3 cluster by downloading the updated the latest `operator.yml` file and replacing the existing Cluster Application Migration Operator CR object.
```
Comment (lamek) [inline modules/migration-upgrading-migration-tool.adoc:34]: "the updated the latest"
I'm guessing you just want "the latest" here.

**Review by lamek — COMMENTED** [review]
LGTM. A few quick nits.

What do you think about "Upgrading" info being in a section titled "Deploying CAM"? I would update the assembly title to "Deploying and **upgrading**...", to give a little extra context, but you can make the call.

File: modules/migration-upgrading-migration-tool.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-upgrading-migration-tool.adoc:17]: > LGTM. A few quick nits.
> 
> What do you think about "Upgrading" info being in a section titled "Deploying CAM"? I would update the assembly title to "Deploying and **upgrading**...", to give a little extra context, but you can make the call.

Only problem is that I will be adding a similar section to 4.5 and I don't want to deal with merge conflicts. Oh, well, I guess I can deal with that when it happens.

**Comment by apinnick** [comment]
@lamek  I made the changes you requested. Ready for merge. Thanks.

**Pull request merged**